### PR TITLE
Improve RunScriptOperation exception message on missing ScriptEngine MC-1120

### PR DESCRIPTION
**Pull request by zhmaeff**
Since there is no bundled JavaScript ScriptEngine available for JDK 15 and later, groovy ScriptEngine was used for tests as it is already in the classpath.

Also improves RunConsoleCommandOperation and RunScriptOperation test coverage.

Checklist:
- [x] Labels (`Team:`, `Type:`, `Source:`, `Module:`) and Milestone set
- [x] Label `Add to Release Notes` or `Not Release Notes content` set
- [ ] Request reviewers if possible

### CODE REVIEW

**Comment by zhmaeff** [comment]
run-with-jdk17

**Comment by zhmaeff** [comment]
run-with-jdk17

**Comment by zhmaeff** [comment]
run-lab-run

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        ClientInvocationFuture future = invocation.invoke();
+        try {
+            future.get(ASSERT_TRUE_EVENTUALLY_TIMEOUT, SECONDS);
+            fail("Execution was successful whereas failure was expected.");
+        } catch (ExecutionException e) {
+            Throwable cause = e.getCause();
+            assertTrue("Cause is of type " + cause.getClass().toString(),
+                    cause.getClass().isAssignableFrom(AccessControlException.class));
+            assertEquals(expectedExceptionMsg, cause.getMessage());
+        }
```
Comment (ldziedziul) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java:93]: Maybe use assertj instead? 

```suggestion
        Assertions.assertThatThrownBy(() -> future.get(ASSERT_TRUE_EVENTUALLY_TIMEOUT, SECONDS))
                .hasCauseInstanceOf(AccessControlException.class)
                .hasRootCauseMessage(expectedExceptionMsg);
```

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    HazelcastClientInstanceImpl client;
+    HazelcastInstance member;
+    private TestHazelcastFactory factory;
+
+    @Before
+    public void setUp() {
+        factory = new TestHazelcastFactory();
+
+        // scripting and console are disabled by default
+        member = factory.newHazelcastInstance(smallInstanceConfig());
```
Comment (ldziedziul) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java:61]: `member` variable is not used
```suggestion
        factory.newHazelcastInstance(smallInstanceConfig());
```

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class DisabledOperationsTest {
+
+    HazelcastClientInstanceImpl client;
```
Comment (ldziedziul) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/DisabledOperationsTest.java:52]:
```suggestion
    private HazelcastClientInstanceImpl client;
```

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunConsoleCommandOperationTest.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+/**
+ * Tests possibility to disable scripting on members.
```
Comment (promanenko) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunConsoleCommandOperationTest.java:40]: Leftover from `ScriptingProtectionTest`

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunScriptOperationTest.java [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+import java.util.concurrent.ExecutionException;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * Tests possibility to disable scripting on members.
```
Comment (promanenko) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunScriptOperationTest.java:46]: the same as in ConcoleTest

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunConsoleCommandOperationTest.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+ */
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class RunConsoleCommandOperationTest extends HazelcastTestSupport {
+
+    HazelcastInstance hz;
```
Comment (ldziedziul) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunConsoleCommandOperationTest.java:46]:
```suggestion
    private HazelcastInstance hz;
```

File: hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunScriptOperationTest.java [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+ */
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class RunScriptOperationTest extends HazelcastTestSupport {
+
+    // Let's use Groovy on classpath, because Azul Zulu 6-7 doesn't include the JavaScript engine (Rhino)
+    // and Nashorn was removed from JDK 15+
+    private static final String GROOVY_ENGINE = "groovy";
+
+    HazelcastInstance hz;
```
Comment (ldziedziul) [inline hazelcast/src/test/java/com/hazelcast/internal/management/operation/RunScriptOperationTest.java:56]:
```suggestion
    private HazelcastInstance hz;
```

**Comment by ldziedziul** [comment]
run-with-jdk17

**Review by ldziedziul — APPROVED** [review]
Thanks, looks good!

**Review by promanenko — APPROVED** [review]

**Review by mdumandag — APPROVED** [review]

**Pull request merged**